app/src/main/res/values/styles.xml

```diff
@@ -3,9 +3,9 @@
     <!-- Base application theme. -->
     <style name="AppTheme" parent="Theme.MaterialComponents.Light.NoActionBar">
         <!-- Customize your theme here. -->
-        <item name="colorPrimary">@color/colorPrimary</item>
-        <item name="colorPrimaryDark">@color/colorPrimaryDark</item>
-        <item name="colorAccent">@color/colorAccent</item>
+        <item name="colorPrimary">#1E2A47</item>
+        <item name="colorPrimaryDark">#172441</item>
+        <item name="colorAccent">#32CD32</item>
     </style>
```

Comment:
You can also add the others colors from the `colors.xml` in the same style here.